airlock/templatetags/airlocktags.py

```diff
@@ -0,0 +1,18 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def as_csv_data(file_content):
+    rows = file_content.strip().split("\n")
+    data = {
+        "headers": _split_row(rows[0]),
+        "rows": [_split_row(row) for row in rows[1:]],
+    }
+    return data
```

Comment:
Is there any reason not to use the `csv` module here? It's less code and will handle quoted values correctly.
```py
reader = csv.reader(file_content.splitlines(keepends=True))
return {
    "headers": next(reader),
    "rows": list(reader),
}
```

Reply:
For some reason the fact that it wasn't a file at this point meant I didn't - will do that